include/seeds.proposals.hpp

```diff
@@ -176,7 +182,7 @@ CONTRACT proposals : public contract {
       void recover_voice(name account);
       void demote_citizen(name account);
       uint64_t calculate_decay(uint64_t voice);
-      name get_type (name fund);
+      name get_type (const name & fund);
```

Comment:
does this make a difference? just curious...

Reply:
amm no, this small change doesn't make any difference in the overall speed, I mean to use & in the functions makes them faster but that change won't make the difference